Package.swift

```diff
@@ -24,6 +24,11 @@
 
 import PackageDescription
 
+// To use a local installation of ArcGIS Runtime for iOS, uncomment the following lines and also uncomment
+// package depenencies:package:path and comment the depenencies:package:url
```

Comment:
These are not the technical way to refer to the swift functions. You could probably not include this information and make the instructions much more general and it would be acceptable.  Something like:

```suggestion
// This is setup by default to use a hosted version of the ArcGIS Runtime for iOS.
// To use a local installation of ArcGIS Runtime for iOS, uncomment the lines below declaring
// the `localArcGISPackage` constant and use that as the package dependency instead of the
// hosted version.
```